src/properties/property-effects.html

```diff
@@ -190,26 +176,28 @@
    */
   function runNotifyEffects(inst, props, oldProps) {
     // Notify
-    let notified;
-    let notifyEffects = inst[TYPES.NOTIFY];
-    let id = effectUid++;
-    // Try normal notify effects; if none, fall back to try path notification
-    // TODO(kschaaf) This is a hot path which we could avoid if (1) no
-    // "notify" effects AND (2) if we knew there were no paths to notify. This
-    // 2nd piece of info is not currently available but could be added perhaps
-    // in `_setProperty`.
-    for (let prop in props) {
-      if (notifyEffects && runEffectsForProperty(inst, notifyEffects, id,
-        prop, props[prop], oldProps && oldProps[prop])) {
-        notified = true;
-      } else if (notifyPath(inst, prop, props[prop])) {
-        notified = true;
+    let notifyEffects = inst.__notifyEffects;
+    if (notifyEffects) {
+      let notified;
+      let id = effectUid++;
+      // Try normal notify effects; if none, fall back to try path notification
+      // TODO(kschaaf) This is a hot path which we could avoid if (1) no
+      // "notify" effects AND (2) if we knew there were no paths to notify. This
+      // 2nd piece of info is not currently available but could be added perhaps
+      // in `_setProperty`.
+      for (let prop in props) {
+        if (notifyEffects && runEffectsForProperty(inst, notifyEffects, id,
+          prop, props, oldProps)) {
+          notified = true;
+        } else if (notifyPath(inst, prop, props)) {
+          notified = true;
+        }
```

Comment:
The intention is that `notified` latches on if _any_ property in the loop over `props` resulted in notification; this code would re-assign `notified` (potentially back to `false`) each prop.

The only purpose of the `nofified` flag is to prevent a frivolous call to `host_flushProperties`, so I might benchmark eliminating that bookkeeping and see if its worth keeping around though.